README.md

```diff
@@ -156,25 +134,71 @@ Currently our implementation saves the start and end time of each trip as a stri
+**Hint:** If you're hitting a `NoMethodError` for `Time.parse`, be aware that you need to `require 'time'` in order for it to work.
 
-#### 1.2: Aggregate Statistics
+#### 1.2: User Aggregate Statistics
 
-Now that we have data for cost available for every trip, we can do some interesting data processing. Each of these should be implemented as an instance method on `Driver` or `Passenger`.
+Now that we have data for cost available for every trip, we can do some interesting data processing. Each of these should be implemented as an instance method on `User`.
 
-1.  Add an instance method to `Passenger` that will return the _total amount of money_ that passenger has spent on their trips
-1.  Add an instance method to `Passenger` that will return the _total amount of time_ that passenger has spent on their trips
-1.  Add an instance method to `Driver` to calculate that driver's _total revenue_ across all their trips. Each driver gets 80% of the trip cost _after_ a fee of $1.65 is subtracted.
-1.  Add an instance method to `Driver` to calculate that driver's _average revenue per hour_ spent driving, using the above formula for revenue
+1.  Add an instance method, `total_spent`, to `User` that will return the _total amount of money_ that user has spent on their trips
+1.  Add an instance method,  `total_time_spent` to `User` that will return the _total amount of time_ that user has spent on their trips
 
-**All of these methods must have tests.**
+**Each of these methods must have tests.**
 
 ### Wave 2
 
-Our program needs a way to make new trips and appropriately assign a driver and passenger.
+Our program needs a data type to represent Drivers in our service.
+
+We will do this by creating a `Driver` class which inherits from `User`.  A `Driver` will add the following data attributes:
+
+**Attribute**|**Description**
+-----|-----
+vehicle\_identification|The driver's Vehicle Identification Number (VIN Number), Each vehicle identification number should be a specific length to ensure it is a valid vehicle identification number
+driven_trips | A list of trips the user has acted as a driver for.
+status|Indicating availability, a driver's availability should be either `:AVAILABLE` or `:UNAVAILABLE`
+
+**Create tests** to ensure that a `Driver` instance can be created successfully.
+
+#### Updating Trip
+
+To make use of the new `Driver` class we will need to update the `Trip` class to include a reference to the trip's driver.  Add the following attribute to the `Trip` class.
+
+**Attribute**|**Description**
+-----|-----
+driver|The `Driver` for the trip
+
+Each `Trip` instance should also be able to do the following:
+
+**Method**|**Description**
+-----|-----
+driver|retrieve the associated `Driver` instance
+
+#### Loading Drivers
+Update the `TripDispatcher` class to add the following Methods:
```

Comment:
Do students also need to update the existing `TripDispatcher` functionality that `Trip`s loaded in must now have a driver associated with it?